ingest/src/main/java/com/connexta/ingest/transform/TransformResponse.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2019 Connexta, LLC
+ *
+ * Released under the GNU Lesser General Public License version 3; see
+ * https://www.gnu.org/licenses/lgpl-3.0.html
+ */
+package com.connexta.ingest.transform;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import java.util.List;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.http.HttpStatus;
+
+@Getter
+@Setter
+public class TransformResponse {
```

Comment:
✏️Use a constructor and final fields instead of setters so that we can enforce that the values are never null and shouldn't be changed